news/2025/2025-02-13-osumania-7k-world-cup-2025-semifinals-recap.md

```diff
@@ -0,0 +1,285 @@
+</div>
+
+![Semifinals mappool table](/wiki/shared/news/2025-02-13-osumania-7k-world-cup-2025-semifinals-recap/mappool.jpg)
+
+We've reached a point where the average Joe may struggle to keep up with the mappool's difficulty. If earlier maps are characterised by their need to get very good accuracy on them, higher-end maps are tests of endurance meant to push players to their limits, where difficulty comes from misses more than anything else. And let us tell you — missing our players will!
+
+Keeping up with the same pool size as last week and its newfound categories, we have several new rehashed maps of [bespoke music](/wiki/Community/Bespoke_music) from around the osu! community in this round's pool! Most interestingly, the first one is [Trina Lydia - TETRATONE ALTiSSiMO](https://osu.ppy.sh/beatmapsets/2317891#mania/4964056) (LN3) mapped by [U1d](https://osu.ppy.sh/users/10125072) and originally released for the [1K Dan Course](https://osu.ppy.sh/beatmapsets/2234936#mania/4747563). That's right, one key! As for returning osu! originals from other World Cups, we have [Lime - Pixel Planet](https://osu.ppy.sh/beatmapsets/2317895#mania/4964061) (RC6) made by [_Kobii](https://osu.ppy.sh/users/6209713), [SiLiS - Hebe](https://osu.ppy.sh/beatmapsets/2317981#mania/4964313) (RC5) by [_Yiiiii](https://osu.ppy.sh/users/6066359), and finally [aquestion - Umami Blaze](https://osu.ppy.sh/users/6066359) (RC1) mapped by [pwhk](https://osu.ppy.sh/users/4887865). Quite the assortment of original rice maps.
+
+But originals are not the only highlights of the pool! As we all know by now, the mini-TB gives players a taste of how an actual tiebreaker is typically designed, both in terms of structure and difficulty. [mitei - Phosphor](https://osu.ppy.sh/beatmapsets/2317893#mania/4964059) (HB3) by [U1d](https://osu.ppy.sh/users/10125072) has it all! Aesthetically pleasing patterns are present throughout the map while covering a wide range of skill sets required to play these patterns effectively. Beyond these challenges, it also offered an enjoyable experience not only for the players but also for the mappool playtesters. The 0% ban rate during the weekend is a testament to that.
+
+On top of that, for the more rice-oriented players, we've got [REDALiCE feat. Ayumi Nomiya - Little Star (Full ver.)](https://osu.ppy.sh/beatmapsets/2317896#mania/4964062) (RC2). Even if it just had a minor increase in BPM as opposed to its Quarterfinals counterpart, the far denser patterns didn't scale as smoothly. This effectively tested players' reading skills and endurance, especially if we consider that it is almost a minute longer from last round's map on the same category. While it might seem daunting to tackle such a map given it was one of the most stamina-intensive picks, it remained a favourite amongst players. So much so that it was the most picked map throughout the Semifinals, with no bans at all!
```

Comment:
```suggestion
On top of that, for the more rice-oriented players, we've got [REDALiCE feat. Ayumi Nomiya - Little Star (Full ver.)](https://osu.ppy.sh/beatmapsets/2317896#mania/4964062) (RC2). Even if it just had a minor increase in BPM as opposed to its Quarterfinals counterpart, the far denser patterns didn't scale as smoothly. This effectively tested players' reading skills and endurance, especially if we consider that it is almost a minute longer from last round's map on the same category. While it might seem daunting to tackle such a stamina-intensive pick, it remained a favourite among players. So much so, that it was the most picked map throughout the Semifinals, with no bans at all!
```